### PR TITLE
Header cleaning in ~~the spring~~ February

### PR DESCRIPTION
**Pull request by rastersize**
- Move framework version constants to top of file.
- Move SPTPersistentCacheChooseKeyCallback to header where it’s used.

@spotify/objc-dev

### CODE REVIEW

**Comment by 8W9aG** [comment]
:+1:

File: include/SPTPersistentCache/SPTPersistentCache.h [hunk 1]
```diff
@@ -20,6 +20,17 @@
...
+#define SPT_BUILDING_FRAMEWORK 0
+#endif
+#if SPT_BUILDING_FRAMEWORK
+//! Project version number for SPTDataLoader.
+FOUNDATION_EXPORT double SPTDataLoaderVersionNumber;
+
+//! Project version string for SPTDataLoader.
+FOUNDATION_EXPORT const unsigned char SPTDataLoaderVersionString[];
+#endif // SPT_BUILDING_FRAMEWORK
+
```
Comment (dflems) [inline include/SPTPersistentCache/SPTPersistentCache.h:33]: What's the purpose of these? (for my own knowledge)

Comment (rastersize) [inline include/SPTPersistentCache/SPTPersistentCache.h:33]: @dflems Those exports are part of what Xcode uses for creating a Framework module and Swift compatibility. Exactly what Xcode uses them for is beyond my knowledge. However we don’t want them for our static library.

Comment (rastersize) [inline include/SPTPersistentCache/SPTPersistentCache.h:33]: You’ll get them as soon as you create a dynamic framework target.

**Pull request merged**